tests/mocks/items.ts

```diff
@@ -0,0 +1,28 @@
+export interface Character {
```

Comment:
do we need this interface only for tests?
Even if yes, I would suggest to put it in a types.ts file

Reply:
Those interfaces are used only for tests, i don't see the value to put them in the `src/`